<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DBExportWidget</class>
 <widget class="QWidget" name="DBExportWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>179</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="exp_groupBox">
     <property name="title">
      <string>Export</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::ExpandingFieldsGrow</enum>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="exp_label_dir">
        <property name="text">
         <string>Export directory</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <layout class="QHBoxLayout" name="exp_horizontalLayout_dir">
        <item>
         <widget class="QLineEdit" name="exp_line_dir"/>
        </item>
        <item>
         <widget class="QPushButton" name="exp_push_dir">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="exp_label_filename">
        <property name="text">
         <string>Export filename</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="exp_line_filename">
        <property name="toolTip">
         <string>Filename for exported database file/archive (doesn't include file extension)</string>
        </property>
        <property name="placeholderText">
         <string>export filename</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="exp_label_info">
        <property name="text">
         <string>Export .info file</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QCheckBox" name="exp_chb_info">
        <property name="toolTip">
         <string>Export INFO file together with database file. This option is available only if database is exported to archive.</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="exp_label_output">
        <property name="text">
         <string>Output filename</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="exp_line_output">
        <property name="toolTip">
         <string>Displays final export file name.</string>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="placeholderText">
         <string>output full filename</string>
        </property>
       </widget>
      </item>
     </layout>
     <zorder>exp_label_dir</zorder>
     <zorder></zorder>
     <zorder>exp_label_filename</zorder>
     <zorder>exp_line_filename</zorder>
     <zorder>exp_label_info</zorder>
     <zorder>exp_chb_info</zorder>
     <zorder>exp_label_output</zorder>
     <zorder>exp_line_output</zorder>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="exp_horizontalLayout_buttons">
     <item>
      <widget class="QPushButton" name="exp_push_export">
       <property name="toolTip">
        <string>Export database to given file</string>
       </property>
       <property name="text">
        <string>&amp;Export</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="exp_push_reset">
       <property name="toolTip">
        <string>Reset to default settings</string>
       </property>
       <property name="text">
        <string>&amp;Reset</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
